index.html

```diff
@@ -293,21 +293,32 @@ <h3>Skills</h3>
           <li>
             <input
               type="text"
-              name=""
-              id=""
+              name="name"
+              id="name"
               placeholder="Full name"
               required
               maxlength="30"
             />
           </li>
 
           <li>
-            <input type="email" placeholder="Your email address" required />
+            <input
+              type="email"
+              id="email"
+              name="email"
+              placeholder="Your email address"
+              required
+            />
           </li>
           <li>
-            <textarea name="" id="" cols="30" rows="10">
-Enter text here</textarea
-            >
+            <textarea
+              name="message"
+              id="message"
+              cols="30"
+              rows="10"
+              placeholder="Enter text here"
+              required
+            ></textarea>
```

Comment:
- [ Optional ] As stated the project requirement Kindly make sure that Message field accepts up to 500 chars.
 
![Capture](https://user-images.githubusercontent.com/93598189/212875994-c5c96ddc-a151-429e-a276-4c9934d419c6.PNG)